OpenRA.Game/FileSystem/ZipFile.cs

```diff
@@ -206,9 +206,10 @@ public Stream GetSource()
 
 		public bool TryParsePackage(Stream s, string filename, FileSystem context, out IReadOnlyPackage package)
 		{
-			var signature = s.ReadUInt32();
+			var test = s.ReadUInt32();
```

Comment:
These changes were squashed into the wrong commit (case insensitive resolving instead of ZipFile). But actually I disagree with the original premise, `signature` is a much clearer name than `test` and the original code was clear enough for anyone who's spent any time with file parsing code.

Reply:
It's not about understanding what the code is doing, as much as it is that Signature != signature is inherently confusing no matter how experienced you are at file parsing code. if (inputSignature != TargetSignature) for example would represent the same code, but be much easier for a dev to parse (no pun intended) and not make a mistake in doing so (since it would be all too easy to confuse a lowercase S with an uppercase one).